index.js

```diff
@@ -75,6 +75,9 @@ module.exports = {
     // require trailing commas
     // https://eslint.org/docs/rules/comma-dangle
     'comma-dangle': [1, 'always-multiline'],
+    // require at least one whitespace after comments( // and /*)
+    // https://eslint.org/docs/rules/spaced-comment
+    'spaced-comment': [1, "always"],
```

Comment:
Sorry, still double quotes here. `npm run lint -- --fix`

Reply:
omg you're right. sorry.